package.json

```diff
@@ -47,6 +48,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify --napi --strip",
+    "download-prebuilds": "rimraf prebuilds && prebuildify-ci download",
```

Comment:
Hm, if I'm reading [this](https://github.com/mafintosh/prebuildify-ci/blob/9810c562ca0252d82454aa109dee878a23896033/index.js#L66) right, prebuildify already cleans the directory, so `rimraf` should not be necessary.